modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-fragment/pagination.spec.ts

```diff
@@ -119,8 +119,8 @@ test.describe(
 						state: 'visible',
 					});
 
-					expect(
-						await fdsFragmentPage.fdsTableWrapper
+					await expect(
+						fdsFragmentPage.fdsTableWrapper
```

Comment:
@juanjofgliferay here I just applied the general pattern of `await expect()` when there is a locator involved. Some other `expect()` calls were left as they have nothing to do with locator resolution. I also removed some unneeded `await`as IDE complained